osu.Game/Screens/OnlinePlay/Lounge/Components/DrawableRoomParticipantsList.cs

```diff
@@ -51,42 +55,99 @@ private void load(OverlayColourProvider colours)
                 },
                 new FillFlowContainer
                 {
-                    Anchor = Anchor.Centre,
-                    Origin = Anchor.Centre,
-                    AutoSizeAxes = Axes.Both,
-                    Direction = FillDirection.Horizontal,
-                    Spacing = new Vector2(4),
-                    Padding = new MarginPadding { Right = 16 },
+                    RelativeSizeAxes = Axes.Y,
+                    AutoSizeAxes = Axes.X,
                     Children = new Drawable[]
                     {
-                        new SpriteIcon
+                        new FillFlowContainer
                         {
-                            Anchor = Anchor.CentreLeft,
-                            Origin = Anchor.CentreLeft,
-                            Size = new Vector2(16),
-                            Margin = new MarginPadding { Left = 8 },
-                            Icon = FontAwesome.Solid.User,
+                            RelativeSizeAxes = Axes.Y,
+                            AutoSizeAxes = Axes.X,
+                            Spacing = new Vector2(8),
+                            Padding = new MarginPadding
+                            {
+                                Left = 8,
+                                Right = 16
+                            },
+                            Children = new Drawable[]
+                            {
+                                hostAvatar = new CircularAvatar
+                                {
+                                    Anchor = Anchor.CentreLeft,
+                                    Origin = Anchor.CentreLeft,
+                                },
+                                hostText = new LinkFlowContainer
+                                {
+                                    Anchor = Anchor.CentreLeft,
+                                    Origin = Anchor.CentreLeft,
+                                    AutoSizeAxes = Axes.Both,
+                                    Text = "hosted by smoogipoo"
```

Comment:
name *probably* shouldn't be hardcoded like that, even if it'll get removed on load complete - I'd leave blank